sonar-project.properties

```diff
@@ -1,7 +1,7 @@
 sonar.projectKey=microsoftgraph_microsoft-graph-toolkit
 sonar.organization=microsoftgraph2
 sonar.projectName=microsoft-graph-toolkit
-sonar.projectVersion=4.2.1
+sonar.projectVersion=4.2.2
```

Comment:
is the intent here that this should match the latest published version of the libraries? 

if that is the case then we should have some action in our release please workflow that updates this file in the same way that it updates the root package.json and config file.

Perhaps using the generic file updater https://github.com/googleapis/release-please/blob/main/docs/customizing.md#updating-arbitrary-files

Reply:
@gavin, we have an action, :push-release calls the setVersion to update versions project-wide, I've updated it to also update this file in every release.

Reply:
This looks like it should work for us, I guess we'll find out when we get it merged and release-please runs